File: testes-unidade/ex-python-testes/ContaBancaria.py
#!/usr/bin/python
# -*- coding: utf8 -*-

class ContaBancaria:
    def __init__(self, titular, saldo=0):
        self.titular = titular
        self.saldo = saldo

    def depositar(self, valor):
        if valor > 0:
            self.saldo += valor
            print(f'Depósito de R${valor:.2f} realizado com sucesso.')
        else:
            print('O valor do depósito deve ser maior que zero.')

    def sacar(self, valor):
        if valor > 0 and valor <= self.saldo:
            self.saldo -= valor
            print('Saque de R${valor:.2f} realizado com sucesso.'.format(
                valor=valor))
        elif valor <= 0:
            print('O valor do saque deve ser maior que zero.')
        else:
            print('Saldo insuficiente.')

    def consultar_saldo(self):
        print(f'Saldo disponível: R${self.saldo:.2f}')

File: testes-unidade/ex-java-testes/ContaBancaria.java
public class ContaBancaria {
    private String titular;
    private double saldo;

    public ContaBancaria(String titular, double saldo) {
        this.titular = titular;
        this.saldo = saldo;
    }

    public void depositar(double valor) {
        if (valor > 0) {
            saldo += valor;
            System.out.println("Depósito de R$" + valor + " realizado com sucesso.");
        } else {
            System.out.println("O valor do depósito deve ser maior que zero.");
        }
    }

    public void sacar(double valor) {
        if (valor > 0 && valor <= saldo) {
            saldo -= valor;
            System.out.println("Saque de R$" + valor + " realizado com sucesso.");
        } else if (valor <= 0) {
            System.out.println("O valor do saque deve ser maior que zero.");
        } else {
            System.out.println("Saldo insuficiente.");
        }
    }

    public double getSaldo() {
        return saldo;
    }
}
